app/javascript/error-handling/index.js

```diff
@@ -0,0 +1,155 @@
+/*
+  Function used to create the skeleton code for the GOV.UK-styled
+  summary errors element
+ */
+createSummaryErrorsSkeleton = function() {
+
+    var summaryErrorsElement = document.getElementById("summary-errors");
+
+    addSummaryErrorsClassAndAttributes(summaryErrorsElement);
+
+    if (summaryErrorsElement.innerHTML.includes("h2") === false) {
```

Comment:
```suggestion
    if (!summaryErrorsElement.innerHTML.includes("h2")) {
```
I _think_ this is will be functionally equivalent and a bit more concise?